### ISSUE TITLE
[mobilvennlighet] - Først og fremst få menyen og tabeller til å se bra ut. på mindre skjermer

### ISSUE THREAD

**Issue by helemork** [Artsdatabanken/data.artsdatabanken.no#16]

**Comment by runningbones** [comment]
Enig i at alle løsninger bør fungere på skjermer av ulik størrelse (responsiv), men kanskje ikke kritisk for en datakatalog?

**Comment by helemork** [comment]
Jo, det vil jeg si det er. Når jeg sier "ser bra ut" mener jeg basically bare få det til å være innafor skjermen.
Det må være universelt lesbart siden det er påkrevd at ting skal være mot universell tilnerming osv. 
Dette er uansett noe som er greit å styre med og i stor grad i boks, men laget punktet for å følge opp meg selv når jeg legger til nye ting :) 

(sidebarmenyen derimot se ubrukelig på mobil nå, men venter med det til vi har funnet ut hvordan alt skal være)
